src/modules/Kdump.rb

```diff
@@ -880,6 +881,100 @@ def Summary
       deep_copy(result)
     end
 
+    # Returns available space for Kernel dump according to KDUMP_SAVEDIR option
+    # only local space is evaluated (starts with file://)
+    #
+    # @return [Integer] free space in bytes or nil if filesystem is not local or no
+    #                   packages proposal is made yet
+    def free_space_for_dump
+      kdump_savedir = @KDUMP_SETTINGS.fetch("KDUMP_SAVEDIR", "file:///var/log/dump")
+      log.info "Using savedir #{kdump_savedir}"
+
+      if kdump_savedir.start_with?("/")
+        log.warn "Using old format"
+      elsif kdump_savedir.start_with?("file://")
+        kdump_savedir.sub!(/file:\/\//, "")
+      else
+        log.info "KDUMP_SAVEDIR #{kdump_savedir.inspect} is not local"
+        return nil
+      end
+
+      # unified format of directory
+      kdump_savedir = File.join("/", kdump_savedir)
+
+      partitions_info = SpaceCalculation.GetPartitionInfo()
```

Comment:
Side note, not your yak to shave, but the method is poorly documented: http://www.rubydoc.info/github/yast/yast-packager/master renders empty, and https://github.com/yast/yast-packager/blob/5b3a37da3f62fed4f9e6e296efc1785db9491288/src/modules/SpaceCalculation.rb#L780 does not mention the units (spoiler: kilobytes).